packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts

```diff
@@ -72,31 +126,103 @@ export async function forceLightningLwcPreview(sourceUri: vscode.Uri) {
     return;
   }
 
+  const platformSelection = await vscode.window.showQuickPick(platformInput, {
+    placeHolder: nls.localize('force_lightning_lwc_preview_platform_selection')
+  });
+  if (!platformSelection) {
+    console.log(`${logName}: No valid selection made for preview...`);
+    return;
+  }
+
   const fullUrl = `${DEV_SERVER_PREVIEW_ROUTE}/${componentName}`;
+  const desktopSelected = platformSelection.id === PreviewPlatformType.Desktop;
+  let target: string = platformSelection.defaultTargetName;
+  if (!desktopSelected) {
+    let placeholderText = nls.localize(
+      'force_lightning_lwc_preview_target_default'
+    );
+    const deviceConfig = getWorkspaceSettings().get('rememberDevice') || false;
```

Comment:
Can we rename `deviceConfig` to something more specific?  `rememberDeviceConfigured` perhaps?